cmd/sequelOverlap/intervalQuery.go

```diff
@@ -71,49 +71,3 @@ func writeToFile(answerChan <-chan *queryAnswer, outfile *fileio.EasyWriter, mer
 		}
 	}
 }
-
```

Comment:
I think you forgot to get rid of the imported packages at the top when you removed these function. Can you make sure to run `go build` to make sure the CMD complies with no errors?

Reply:
This now compiles as well in the new commit.